tests/functional/r/redefined_builtin_allowed.rc

```diff
@@ -0,0 +1,2 @@
+[variables]
+allow-redefined-builtins = dir
```

Comment:
Shouldn't it be "allowed" instead ?

Reply:
Agreed. Replaced to "allowed" everywhere.